### ISSUE TITLE
Missing keyboard navigation support

### ISSUE THREAD

**Issue by anondeveg** [casimir/frigoligo#341]
There is no way to scroll down using the keyboard—only the mouse works. I'm not sure if this is a bug or if arrow key support is missing, but the application is difficult to use without a keyboard-based scrolling option.

Could you confirm if this is intended behavior or a bug?

Thanks!

**Comment by casimir** [comment]
There's not particular support of keyboard shortcuts as of new. What kind of support would you expect? Scrolling on the reading screen, navigation in the whole application?

**Comment by anondeveg** [comment]
deally, the entire application should support navigation using the arrow keys, especially on the reading screen.
